AT-0001-DP Modal en home page

Angelportillap opened the pull request:
Se crea modal de cerrar sesión y editar usuario

Angelportillap committed:
AT-0001-DP Modal en home page

sebastianpl1306 commented on src/app/components/home-page/HomePage.vue:
```diff
@@ -29,22 +29,62 @@
           <a class="navbar-brand text-white" href="">UsuarioAdmin</a>
         </div>
         <div action="" class="d-flex p-3">
-          <a class="navbar-brand text-white">Cerrar sesion</a>
```
se deberia usar la libreria font-awesome

sebastianpl1306 commented on src/app/components/home-page/HomePage.vue:
```diff
@@ -29,22 +29,62 @@
+              <h5 class="text-white">Editar perfil</h5>
+              <button type="button" class="btn-close text-white" aria-label="Close"></button>
+            </div>
+            <div class="modal-body row aling-content-center justify-content-center  col-8 m-2">
```
agrandar el col-8 a col-12 para que ocupe toda la modal

sebastianpl1306 commented on src/app/components/home-page/HomePage.vue:
```diff
@@ -111,41 +152,57 @@
-                    </div>
-                  </div>
+                  <table class="table m-0 justify-content-center">
+                    <!-- <thead class="">
```
No deberia haber codigo comentado

sebastianpl1306 commented on src/app/components/home-page/HomePage.vue:
```diff
@@ -111,41 +152,57 @@
 </template>
-
-<script src="./homePage.ts" lang="ts"></script>
+<!-- <template class="Router">
```
No deberia haber codigo comentado

Angelportillap committed:
AT-0001-DP  Se aregla incidencias PR

Se amplia el tamaño de la informacion del modal editUser, se elimina el codigo comentado
